### ISSUE TITLE
FPS drop when many ships collied with ground

### ISSUE THREAD

**Issue by BigBadKangaroo** [tsunamayo/Starship-EVO#4371]
Hi,

When I am at the Torus I usually get around 180-200 FPS with nothing spawned (no ships, no buildings)
The more ships I spawn that are pulled down by gravity of the Torus (touching the ground) the less FPS I get.
They are even going down to around 50-60, even when I don't look at them but when I'm near to them.
Large ships can sometimes even get me down to 30-40 FPS.

Collision seems to hit the FPS quite hard regarding the Torus.

*all of the screenshots are with NPC's disabled!*

![20211217223440_1](https://user-images.githubusercontent.com/81495956/146610843-84ec8397-8d7d-4dbc-a5a7-5504fddcde06.jpg)

There are not many ships, and they are all smaller builds!
![20211217223645_1](https://user-images.githubusercontent.com/81495956/146610859-80e27d4c-69d7-4495-8fc7-d6c293860ef6.jpg)

This is with the spawned ships but not looking at them
![20211217223656_1](https://user-images.githubusercontent.com/81495956/146610866-6894106b-5bcf-4c20-b0e6-82ec450fe3a0.jpg)

![20211217223837_1](https://user-images.githubusercontent.com/81495956/146611022-541d62a9-759d-43ad-9787-9d40674d476f.jpg)

**Comment by tsunamayo** [comment]
@BigBadKangaroo Yes torus is less optimized and wont be as I will refactor the terrain technology. Does your ship come to rest eventually? Thanks

**Comment by BigBadKangaroo** [comment]
Yes it does. But any type of movement in the ship itself (i.e. self build doors) drop the fps down again. The best way to land the ship on the torus is to place a build landingpad down before landing.
